### DISCUSSION TITLE
Songselect-bottom/top.png overlay

### DISCUSSION THREAD

**Discussion by 7deok7** [ppy/osu#25789] [Ideas] [answered]
Since the goal of the songselect-bottom/top files is to make stable skins compatible with lazer, adding something like a songselect-bottom/top-overlay would be needed. I think this must be added is because a lot of skins, almost every skin with a custom song select, hides the osu! cookie in the bottom right. If it's not added people are still going to use the old method of selection-mode.png as a way to overlay the osu! cookie, which would then result in a skin incompatible with lazer.

You can actually see this in the new skin WhiteCat 3.0, made by cpol. He uses both selection-mode.png and songselect-bottom.png, but heavily relies on selection-mode, due to the reasons I've listed above.

**Comment by peppy** [comment, marked as answer]
song select skinning will come at a later point.
